plugins/modules/aci_l4l7_policy_based_redirect.py

```diff
@@ -0,0 +1,391 @@
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Policy Based Redirection
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  policy_name:
+    description:
+    - The name of the Policy Based Redirection Policy.
+    type: str
+    aliases: [ policy ]
+  description:
+    description:
+    - The description of the Policy Based Redirection Policy.
+    type: str
+  destination_type:
+    description:
+    - The destination type.
+    - The APIC defaults to C(l3) when unset during creation.
+    type: str
+    choices: [ l1, l2, l3 ]
+    aliases: [ dest_type ]
+  hash_algorithm:
+    description:
+    - The hashing algorithm.
+    - The APIC defaults to C(ip_and_protocol) when unset during creation.
+    type: str
+    choices: [ source_ip, destination_ip, ip_and_protocol ]
+  threshold_enable:
+    description:
+    - Whether to enable the threshold for the policy.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  max_threshold:
+    description:
+    - The maximum percent when threshold is enabled.
+    - The APIC defaults to C(0) when unset during creation.
+    type: int
+  min_threshold:
+    description:
+    - The minimum percent when threshold is enabled.
+    - The APIC defaults to C(0) when unset during creation.
+    type: int
+  threshold_down_action:
+    description:
+    - The action to take when threshold is breached.
+    - The APIC defaults to C(permit) when unset during creation.
+    type: str
+    choices: [ deny, permit ]
+  resilient_hash:
+    description:
+    - Whether to enable resilient hashing.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  pod_aware:
+    description:
+    - Whether to enable Pod ID aware redirection.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  anycast_enabled:
+    description:
+    - Whether to enable anycast services.
+    - The APIC defaults to C(false) when unset during creation.
+    - Only available when I(destination_type=l3)
+    type: bool
+  ip_sla_monitor_policy:
+    description:
+    - The name of the IP SLA Monitoring Policy to bind to the L4-L7 Redirect Policy.
+    - To remove an existing binding to an IP SLA Monitoring Policy, submit a request with I(state=present) and I(ip_sla_monitor_policy="") value.
+    type: str
+    aliases: [ monitor_policy, sla, sla_policy ]
+  rewrite_source_mac:
+    description:
+    - Whether to rewrite the source MAC address of forwarded traffic.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+notes:
```

Comment:
Shouldn't the `ip_sla_monitor_policy` also exist before using?

Reply:
We only mention the required attributes or the parent object/top object in the notes section